share/completions/boon.fish

```diff
@@ -0,0 +1,47 @@
+set command boon
+
+complete -c $command -f
+
+complete -c $command \
+    -s h \
```

Comment:
Please don't break these lines up so much. Just put it on one line, possibly two.

Reply:
Ig, I can use 80 character limit to determine whether to distribute a line over several ones or not. Wanna keep completions readable and not to make myself scroll horizontally.

Reply:
I will apply the same logic for all other my PRs. 😄

Reply:
These are *well below* 80 characters, which is a limit that we do not actually *have* in our fish script style.

If you must split them up, here's how I would do that:

```fish
complete -c $command \
    -s o \
    -d 'The output file of a migrated schema' \
    -n "__fish_seen_subcommand_from validate migrate" \
    -F -r
```

becomes

```fish
complete -c $command -s o -F -r \
    -n "__fish_seen_subcommand_from validate migrate" \
    -d 'The output file of a migrated schema'
```

Basically put the big condition and description on their own lines, but *don't* give each little option its own line. That just wastes a ton of vertical space and gives you very empty lines (`-s o` is basically nothing).

Reply:
I like your suggestions, I also put `-a` on a separate line, to keep the following structure of the completion:

- option's values
- options with additional flags like `-F`
- description
- condition

Reply:
That's fine.